src/lib/device-types/index.ts

```diff
@@ -12,7 +12,7 @@ import {
 } from './build-info-facade';
 import { getImageKey, IMAGE_STORAGE_PREFIX, listFolders } from './storage';
 
-const { BadRequestError, NotFoundError } = sbvrUtils;
+export const { BadRequestError, NotFoundError } = sbvrUtils;
```

Comment:
Why are these being exported now?

Reply:
TS complains we're extending a private type below.

Reply:
Ah right, because `declaration: true` was added, that kinda sucks because it's not actually private, just not exported by this module